src/main/webapp/resources/paymentMethod/slipDetailsAsOnlyPayment.xhtml

```diff
@@ -21,6 +21,7 @@
                     id="bank"
                     value="#{cc.attrs.paymentMethodData.slip.institution}"
                     styleClass="form-control"
+                    style="width: 100%"
```

Comment:
Instead of using online styles, we prefer to use styleClass="w-100". Remove  style="width: 100%" and add w-100 to styleClass="form-control"